### ISSUE TITLE
Ensure project is conform to the submission criteria before submission

### ISSUE THREAD

**Issue by picost** [picost/udl_tvscript_project#9]
Everything seems to be ok, except the loss could be decreased by more training steps (although there is no prevention against over-fitting here.
-> increase the number of epochs.
